azure-pipelines.yml

```
@@ -24,162 +25,106 @@ trigger:
-            artifactType: Container
-            parallel: true
-        - task: PublishBuildArtifacts@1
-          displayName: Upload package artifacts
-          condition: and(succeeded(), eq(variables['system.pullrequest.isfork'], false), eq(variables['_BuildConfig'], 'Release'))
-          inputs:
-            pathtoPublish: artifacts/packages/
-            artifactName: artifacts
-            artifactType: Container
-            parallel: true
-
-      - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-        - job: Ubuntu
-          displayName: 'Ubuntu 20.04'
-          pool:
-            vmImage: ubuntu-latest
-          variables:
-          - name: _SignType
-            value: none
-          - name: _OfficialBuildArgs
-            value: ''
-
-          steps:
-          - checkout: self
-            clean: true
-          - script: eng/common/cibuild.sh
-              --configuration $(_BuildConfig)
-              --prepareMachine
-            displayName: Build
-          - task: PublishBuildArtifacts@1
-            displayName: Upload TestResults
-            condition: always()
-            continueOnError: true
-            inputs:
-              pathtoPublish: artifacts/TestResults/$(_BuildConfig)/
-              artifactName: $(Agent.Os)_$(Agent.JobName) TestResults
-              artifactType: Container
-              parallel: true
-      
-      # Tests on macOS are unreliable
-      # - job: macOS
-      #   displayName: 'macOS 10.15'
-      #   pool:
-      #     vmImage: macOS-1015
-      #   variables:
-      #   - name: _SignType
-      #     value: none
-
-      #   - ${{ if and(eq(variables['System.TeamProject'], 'internal'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-      #     - name: _OfficialBuildArgs
-      #       value: -p:OfficialBuildId=$(Build.BuildNumber)
-      #   # else
-      #   - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-      #     - name: _OfficialBuildArgs
-      #       value: ''
-      #   steps:
-      #   - checkout: self
-      #     clean: true
-      #   - script: eng/common/cibuild.sh
-      #       --configuration $(_BuildConfig)
-      #       --prepareMachine
-      #     displayName: Build
-      #   - task: PublishBuildArtifacts@1
-      #     displayName: Upload TestResults
-      #     condition: always()
-      #     continueOnError: true
-      #     inputs:
-      #       pathtoPublish: artifacts/TestResults/$(_BuildConfig)/
-      #       artifactName: $(Agent.Os)_$(Agent.JobName) TestResults
-      #       artifactType: Container
-      #       parallel: true
+      - template: /eng/common/templates-official/jobs/jobs.yml@self
+        parameters:
+          enableMicrobuild: true
+          enablePublishBuildArtifacts: true
+          enablePublishTestResults: true
+          enablePublishBuildAssets: false
```

I believe this is what's causing your BAR publishing (and release configs) to not happen in your test builds @DrewScoggins 

```suggestion
          enablePublishBuildAssets: true
```